packages/auth/src/signature/middleware.test.ts

```diff
@@ -294,9 +297,11 @@ describe('Signature Service', (): void => {
         resource_server: 'test'
       }
       await expect(tokenHttpsigMiddleware(ctx, next)).rejects.toMatchObject({
-        status: 400,
-        error: 'invalid_request',
-        message: 'invalid signature headers'
```

Comment:
I've change this and other error responses to `401` as it is a failure to verify the signature. I also chose `invalid_client` as the code because of the following description:
> "invalid_client":
The request was made from a client that was not recognized or allowed by the AS, **or the client's signature validation failed.**

(emphasis mine)